robot/impl/local_robot.go

```diff
@@ -38,6 +40,8 @@ import (
 
 var _ = robot.LocalRobot(&localRobot{})
 
+var resourceConfigurationTimeout = time.Minute
```

Comment:
```suggestion
var defaultResourceConfigurationTimeout = time.Minute
```
[nit] this technically may not be the actual resource configuration timeout.